### DISCUSSION TITLE
Add --theme=dark in startup options

### DISCUSSION THREAD

**Discussion by ostap667inbox** [lllyasviel/Fooocus#952] [Ideas]
Once the WebUI is launched, the light theme is launched by default.
Of course, there is nothing complicated to add manually in the address bar the parameter `?__theme=dark`.
But it has to be done every time. The `--theme=dark` parameter which could be written in run.bat, as it is done, for example, in Automatic1111, is very much missing.

**Comment by lllyasviel** [comment]
added in 2.1.817
